src/newfilewizard/newfilewizard.cpp

```diff
@@ -110,24 +122,30 @@ void NewFileDialog::onCreateFile()
     QString dirName = mpDirLbl->text();
 
     // check if directory is not empty
-    if(dirName.isEmpty())
+    if (dirName.isEmpty())
     {
         QMessageBox::warning(this, "Wrong directory",
```

Comment:
It is better to create constants for the hardcoded text

Reply:
Fixed that